index.js

```diff
@@ -49,4 +49,45 @@ const elementReady = (selector, {
 	return Object.assign(promise, {stop});
 };
 
+elementReady.subscribe = (selector, callback, {
+	target = document,
+	stopOnDomReady = true,
+	timeout = Infinity
+} = {}) => {
+	const seenElements = new WeakSet();
+
+	let rafId;
+	let checkFrame = true;
```

Comment:
This should should have a `should`, `is`, or `has` prefix. Not clear to me which one fits the best.